src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/Http/ApprenticeshipsInnerApiClient.cs

```diff
@@ -2,94 +2,94 @@
-        private DateTime _tokenExpiry;
+namespace SFA.DAS.Funding.SystemAcceptanceTests.Helpers.Http;
 
-        public ApprenticeshipsInnerApiClient()
-        {
-            _fundingConfig = Configurator.GetConfiguration();
+public class ApprenticeshipsInnerApiClient
+{
+    private readonly FundingConfig _fundingConfig;
+    private readonly HttpClient _httpClient;
+    private string _cachedToken;
+    private DateTime _tokenExpiry;
 
-            var baseUrl = _fundingConfig.ApprenticeshipsInnerApiClientBaseUrl;
-            _httpClient = HttpClientProvider.GetClient(baseUrl);
-        }
+    public ApprenticeshipsInnerApiClient()
+    {
+        _fundingConfig = Configurator.GetConfiguration();
+        var baseUrl = _fundingConfig.ApprenticeshipsInnerApiClientBaseUrl;
+        _httpClient = HttpClientProvider.GetClient(baseUrl);
+    }
 
-        public async Task<HttpResponseMessage> PostAsync(string url, object body)
-        {
-            await EnsureBearerToken();
-            return await _httpClient.PostAsJsonAsync(url, body);
-        }
+    public async Task<HttpResponseMessage> PostAsync(string url, object body)
+    {
+        EnsureBearerToken();
+        return await _httpClient.PostAsJsonAsync(url, body);
+    }
 
-        public async Task<HttpResponseMessage> PatchAsync(string url, object body)
-        {
-            await EnsureBearerToken();
-            return await _httpClient.PatchAsJsonAsync(url, body);
-        }
+    public async Task<HttpResponseMessage> PatchAsync(string url, object body)
+    {
+        EnsureBearerToken();
+        return await _httpClient.PatchAsJsonAsync(url, body);
+    }
 
-        private async Task EnsureBearerToken()
+    private void EnsureBearerToken()
+    {
+        if (string.IsNullOrEmpty(_cachedToken) || DateTime.UtcNow >= _tokenExpiry)
         {
-            if (string.IsNullOrEmpty(_cachedToken) || DateTime.UtcNow >= _tokenExpiry)
-            {
-                await AddBearerToken();
-            }
+            AddBearerToken();
         }
+    }
 
-        private async Task AddBearerToken()
-        {
-            var claims = GetClaims();
-            var signingKey = _fundingConfig.ApprenticeshipServiceBearerTokenSigningKey;
+    private void AddBearerToken()
+    {
+        var claims = GetClaims();
+        var signingKey = _fundingConfig.ApprenticeshipServiceBearerTokenSigningKey;
 
-            var accessToken = ServiceBearerTokenProvider.GetServiceBearerToken(signingKey);
+        var accessToken = ServiceBearerTokenProvider.GetServiceBearerToken(signingKey);
 
-            accessToken = BearerTokenHelper.AddClaimsToBearerToken(accessToken, claims, signingKey);
+        accessToken = BearerTokenHelper.AddClaimsToBearerToken(accessToken, claims, signingKey);
 
-            _cachedToken = accessToken;
-            _tokenExpiry = DateTime.UtcNow.AddMinutes(20);
+        _cachedToken = accessToken;
+        _tokenExpiry = DateTime.UtcNow.AddSeconds(100);
```

Comment:
Is this long enough? If we have a test that waits to assert that an unexpected change is not made then calls this inner api might be cutting it fine.